benchmarks/bevt/ootb/Dockerfile.rocm

```diff
@@ -15,7 +15,7 @@ RUN git clone https://github.com/ROCmSoftwarePlatform/apex && \
     cd ..  && rm apex -r
 
 # Install mmcv from source. Cherry-picking a fix.
-RUN git clone https://github.com/open-mmlab/mmcv && \
+RUN git clone -b bevt_branch https://github.com/zstreet87/mmcv && \
```

Comment:
While this works for now, I am concerned that this is unmaintainable. It's not clear how https://github.com/zstreet87/mmcv/tree/bevt_branch was generated, since its 90 commits ahead, and not all of them are yours. 

Cant we pin mmcv to an older version like how BEVT repository pinned ?